src/AutoRest.CSharp/Output/Models/Types/OutputLibrary.cs

```diff
@@ -11,125 +11,35 @@
 
 namespace AutoRest.CSharp.Output.Models.Types
 {
-    internal class OutputLibrary
+    internal abstract class OutputLibrary
     {
-        private readonly CodeModel _codeModel;
-        private readonly BuildContext _context;
+        protected readonly CodeModel _codeModel;
```

Comment:
Let's not make fields protected.

Reply:
Would you rather have this base class code duplicated? Or just wrapper properties?